plugins/modules/system/parted.py

```diff
@@ -548,6 +554,7 @@ def main():
             part_type=dict(type='str', default='primary', choices=['extended', 'logical', 'primary']),
             part_start=dict(type='str', default='0%'),
             part_end=dict(type='str', default='100%'),
+            fs_type=dict(type='str', default='ext4'),
```

Comment:
```suggestion
            fs_type=dict(type='str'),
```
See my comment about the default